src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.MultiAgent.cs

```diff
@@ -87,14 +88,14 @@ private void EnsureWorkerIsRunning()
                     // Create pipe used to forcefully wake up curl_multi_wait calls when something important changes.
                     // This is created here rather than in Process so that the pipe is available immediately
                     // for subsequent queue calls to use.
-                    Debug.Assert(_wakeupRequestedPipeFd == 0, "Read pipe should have been cleared");
-                    Debug.Assert(_requestWakeupPipeFd == 0, "Write pipe should have been cleared");
+                    Debug.Assert(_wakeupRequestedPipeFd == null, "Read pipe should have been cleared");
+                    Debug.Assert(_requestWakeupPipeFd == null, "Write pipe should have been cleared");
                     unsafe
                     {
                         int* fds = stackalloc int[2];
                         while (Interop.CheckIo(Interop.Sys.Pipe(fds))) ;
```

Comment:
I decided not to make this change.  See https://github.com/dotnet/corefx/pull/4791/files#r46606957 for the discussion.